services/wallet/collectibles/controller.go

```diff
@@ -215,12 +216,14 @@ func (c *Controller) startPeriodicalOwnershipFetchForAccountAndChainID(address c
 		c.commands[address] = make(commandPerChainID)
 	}
 
+	receivedCollectiblesCb := c.notifyCommunityCollectiblesReceived
```

Comment:
considering `receivedCollectiblesCb` it is used only once we could probably avoid having it